domains/luis010923.is-a-backend.dev.is-a-backend.dev.json

```
@@ -0,0 +1,14 @@
+{
+    "domain": "is-a-backend.dev",
+    "subdomain": "luis010923.is-a-backend.dev",
```

```suggestion
    "subdomain": "luis010923",
```

Please also remove the second `.is-a-backend.dev` from the file name too.